src/views/LoginView.vue

```diff
@@ -70,17 +137,16 @@ onBeforeMount(() => {
   }
 });
 
-const ldapLogin = async (event: Event) => {
-  event.preventDefault();
-  if (!values.username || !values.password) return;
-  await authStore.gewisLdapLogin(values.username, values.password, apiService).then(() => {
-    console.error("loggedin");
+const ldapLogin = loginForm.handleSubmit(async () => {
+  if (!username.value.modelValue || !password.value.modelValue) return;
```

Comment:
Dit is redundant, immers zijn deze values defined (omdat ze required zijn)

Reply:
Removed redundant check for values in commit https://github.com/GEWIS/sudosos-frontend-vue3/pull/31/commits/c0cd5afae61160be4c481f76a38752281f930aa6